### ISSUE TITLE
Sense double counts Tesla Power when using a 120 volt mobile charger

### ISSUE THREAD

**Issue by jbandito** [israndy/TesSense#5]
First off, thanks for all the work on this.  This thing is awesome and works great out of the box.  I did have one issue:

I'm using the 120v mobile charger so on a 15a circuit I get 12A*120V = 1440 Watts of power for Tesla Charging.

The Tesla side of the code seems correct on voltage.

Level: 53 %, Limit 80 %, 5.0 MPH **114 Volts** 2.74 kWh added,

But then the Tesla bubble in Sense is 1440*2 = ~2880 Watts

My short term hack was to divide the value on line 136 by 2

        volts = int(sense.active_voltage[0] + sense.active_voltage[1])/2

I'm pretty sure the voltage issue is only on the sense side but I'll see what it does as the sun goes down this evening.  If I come up with a reasonable fix (likely using the Tesla voltage number is it seems like it's half of the sense number), I'll send it to you.

**Comment by israndy** [comment]
Yes, sorry, the code is not super generic, mostly it’s just to my situation and my charger is plugged in via 240v

Glad it’s working for you if it is
